test/verify/check-storage-anaconda

```diff
@@ -638,6 +638,29 @@ AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
         self.dialog_wait_close()
         b.wait_text(self.card_row_col("Storage", 1, 3), "Unformatted data")
 
+    @testlib.skipImage("No Stratis", "debian-*", "ubuntu-*")
+    def testNoOndemandPackages(self):
+        b = self.browser
+        m = self.machine
+
+        disk = self.add_loopback_disk(name="loop10")
+
+        m.execute("systemctl stop stratisd && dnf remove -y stratisd stratis")
```

Comment:
This is destructive, so the test [fails everywhere](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20950-12e61ba7-20240828-102021-fedora-40-devel/log.html#395-1) the second time around, and any stratis test that may run after it. Perhaps it's enough to `systemctl mask` the service? Or does it test for a file? If that's too hard, let's just make it destructive (but then preferably add it to an already existing destructive test)

Also, please don't use `&&` in execute() calls. They already are `set -e`, and `&&` has weird semantics.

Reply:
I have made it destructive.